### PR TITLE
I Korolev

### PR DESCRIPTION
**Pull request by Korolev731**

### CODE REVIEW

File: IKorolev/04.Infra.Vagrant/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+> vagrant up
+* Start virtual machine.
+
+> vagrant share
+* Share your virtual machine to the world via a temporary and unique url.
+
+> vagrant halt
+* Halt virtual machine.
+
+
```
Comment (pluhin) [inline IKorolev/04.Infra.Vagrant/README.md:73]: To many empty lines, please remove

File: IKorolev/04.Infra.Vagrant/1/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
```
Comment (pluhin) [inline IKorolev/04.Infra.Vagrant/1/Vagrantfile:1]: Why did you add this file?

File: IKorolev/04.Infra.Vagrant/Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
```
Comment (pluhin) [inline IKorolev/04.Infra.Vagrant/Vagrantfile:1]: Why did you add this file?

**Review by pluhin — CHANGES_REQUESTED** [review]

File: IKorolev/04.Infra.Vagrant/README.md (hunk 1 above)
Comment (Korolev731) [reply IKorolev/04.Infra.Vagrant/README.md]: fixed

File: IKorolev/04.Infra.Vagrant/1/Vagrantfile (hunk 2 above)
Comment (Korolev731) [reply IKorolev/04.Infra.Vagrant/1/Vagrantfile]: fixed

File: IKorolev/04.Infra.Vagrant/Vagrantfile (hunk 3 above)
Comment (Korolev731) [reply IKorolev/04.Infra.Vagrant/Vagrantfile]: fixed

**Review by pluhin — APPROVED** [review]
Good, approved
